lib/active_stash/stash_indexes.rb

```diff
@@ -157,7 +157,8 @@ def build!
 
     def fields
       fields = @model.attribute_types.inject({}) do |attrs, (k,v)|
-        attrs.tap { |a| a[k] = v.type }
+        type = v.class.name == "ActiveRecord::Encryption::EncryptedAttributeType" ? v.cast_type.type : v.type
```

Comment:
I reckon you can do:
```ruby
type = ActiveRecord::Encryption::EncryptedAttributeType === v ? v.cast_type.type : v.type
```

Reply:
ah nice, that looks heaps better. I'll update.